locales-uk-UA.xml

```diff
@@ -240,16 +605,30 @@
     <term name="interviewer" form="verb">інтерв'ю</term>
     <term name="recipient" form="verb">для</term>
     <term name="reviewed-author" form="verb">рецензія від</term>
+    <term name="collection-editor" form="verb-short">ed. by</term>
     <term name="translator" form="verb">перекладено</term>
     <term name="editortranslator" form="verb">відредаговано й перекладено</term>
 
     <!-- SHORT VERB ROLE FORMS -->
+    <term name="compiler" form="verb-short">comp. by</term>
+    <term name="contributor" form="verb-short">w.</term>
+    <term name="curator" form="verb-short">cur. by</term>
+    <term name="executive-producer" form="verb-short">exec. prod. by</term>
+    <term name="guest" form="verb-short">w. guest</term>
+    <term name="host" form="verb-short">hosted by</term>
+    <term name="narrator" form="verb-short">narr. by</term>
+    <term name="organizer" form="verb-short">org. by</term>
+    <term name="performer" form="verb-short">perf. by</term>
+    <term name="producer" form="verb-short">prod. by</term>
+    <term name="script-writer" form="verb-short">writ. by</term>
+    <term name="series-creator" form="verb-short">cre. by</term>
     <term name="director" form="verb-short">реж.</term>
     <term name="editor" form="verb-short">зе ред.</term>
     <term name="editorial-director" form="verb-short">за заг. ред.</term>
     <term name="illustrator" form="verb-short">іл.</term>
     <term name="translator" form="verb-short">пер.</term>
-    <term name="editortranslator" form="verb-short">ред. і пер.</term>
+    <term name="editortranslator" form="verb-short">ed. &amp; trans. by</term>
+>>>>>>> master
```

Comment:
@POBrien333 you missed one of these conflict markers. This usually happens is you manually edit a conflict vs. click the "Accept … changes" options. In those cases, be sure to double check

Reply:
That'd be great. Thanks :)
Appreciate you looking over this in more detail.